projects/kit/src/index.ts

```diff
@@ -3,6 +3,7 @@ export {maskitoDateRangeOptionsGenerator} from './lib/masks/date-range';
 export {maskitoDateTimeOptionsGenerator} from './lib/masks/date-time';
 export {maskitoNumberOptionsGenerator, maskitoParseNumber} from './lib/masks/number';
 export {maskitoTimeOptionsGenerator} from './lib/masks/time';
+export {maskitoCaretGuard, maskitoEventHandler, maskitoRejectEvent} from './lib/plugins';
```

Comment:
Should we name all plugin with `Plugin` suffix?

```suggestion
export {
    maskitoCaretGuardPlugin,
    maskitoEventHandlerPlugin,
    maskitoRejectEventPlugin
} from './lib/plugins';
```

Reply:
Some of them are generators of plugins, I think it would make name too long to add both so let's just keep it as is.